_data/materials.yaml

```diff
@@ -42,3 +42,8 @@
 - id: a4
   name: DIY Linux Post-Install
   slides: https://docs.google.com/presentation/d/1r3pl2XccayoUaKCFtG3wF6N_qun8VVw7jH5MEvVTBSw/edit
+
+- id: b4
```

Comment:
`b5`, not `b4` (an easy mistake, I did this when I copied and pasted for this one lol)